docs/part3.md

```diff
@@ -13,7 +13,41 @@ terraform init
 
 Expected output:
 
-<!-- SCREENSHOT -->
+```shell
+terraform init
```

Comment:
```suggestion
$ terraform init
```

Reply:
$ prefix is conventional for markdown examples where the commands and output are given

Reply:
https://cirosantilli.com/markdown-style-guide/#dollar-signs-in-shell-code